keyboards/runes/skjoldr/config.h

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright 2021 Vicktor
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0x726E
+#define PRODUCT_ID   0x736B
+#define DEVICE_VER   0x0001
+#define MANUFACTURER Runes
+#define PRODUCT      Skjöldr
```

Comment:
Does this work properly? Unicode characters may not work properly due to the fact that these defines aren't string literals (yet).

Reply:
Sorry for the super late reply. I tested this with this keyboard and my previous one ([VÆNGR](https://github.com/qmk/qmk_firmware/blob/master/keyboards/runes/vaengr/config.h)) and I don't have any problem. 

I have tested and it work fine on Windows, macOS and Linux.

It was recognized properly in linux lsusb and windows control panel:
![image](https://user-images.githubusercontent.com/9425865/177026106-1ad75e17-f38c-44d7-9b0a-bfb699d4923f.png)

Once again, really sorry for the late reply. Thought I already replied to this.